iris-backend-service/src/main/resources/application-dev_env.properties

```diff
@@ -8,3 +8,6 @@ iris.messages.zammad.environment=empty
 iris.messages.github.personal-access-token=DISABLED
 iris.messages.github.repo-org=empty
 iris.messages.github.repo-name=empty
+
+iris.tokens.datSalt=${random.value}
```

Comment:
If we set it random, there is no possibility to test? As long as the used salt is not written to logs.

Reply:
I thought so too, but you can always overwrite the properties from the outside (system property or environment variable).

Reply:
I got a warning after the push that a secret is checked in here. Even though this is one for Dev, I removed it this way.